README.md

```diff
@@ -308,6 +308,42 @@ There are two possible workarounds for this:
 
   </details>
 
+### Using MacOS after removing `nix` while `nix-darwin` was still installed, network requests fail
+
+If `nix` was previously uninstalled (using the [reference documentation](https://nixos.org/manual/nix/stable/installation/uninstall))
+but `nix-darwin` was not uninstalled beforehand, users may experience errors similar to this:
+
+```bash
+$ nix profile install nixpkgs#curl
```

Comment:
Unlikely that users will use this, but we probably shouldn't suggest making ~permanent changes when a simple `nix shell` will also reproduce the issue:

```suggestion
$ nix shell nixpkgs#curl
```